package/koreader/KOReader.oxide

```diff
@@ -0,0 +1,12 @@
+{
+  "displayName": "KOReader",
+  "description": "ebook reader",
```

Comment:
Would be best to sync this with the package description.
```suggestion
  "description": "An ebook reader application supporting PDF, DjVu, EPUB, FB2 and many more formats",
```

Reply:
Should this also be updated in the draft file? I copied from there.

Reply:
Hah, never noticed that. I guess it’s as good a time as any to fix this. Thanks! :)